src/uct/ib/base/ib_iface.c

```diff
@@ -205,6 +205,28 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    "Reverse Service level. 'auto' will set the same value of sl\n",
    ucs_offsetof(uct_ib_iface_config_t, reverse_sl), UCS_CONFIG_TYPE_ULUNITS},
 
+  {"SEND_PRE_OVERHEAD", UCS_VALUE_AUTO_STR,
+   "Time spent to prepare message request and pass it to the hardware or\n"
```

Comment:
This is IB config - so maybe give it more specific names
"SEND_OVERHEAD", .., "Estimated overhead of preparing a work request and posting it to the NIC"

Reply:
Added configuration option: `SEND_OVERHEAD`.